### PR TITLE
4) Dashboard v2 - Resources Card

### PR DESCRIPTION
**Pull request by LukasStordeur**
# Description

Stage 4 of the dashboard v2. 
The resources card was added.
Clicking on a section on the progressbars will navigate to the resources page with the filter preset. The !orphan
is by default kept when redirecting. 

https://github.com/user-attachments/assets/b8591e75-9837-45e9-b53e-0524b91f059b

closes: 
- #7139

### CODE REVIEW

File: src/Slices/DashboardV2/UI/resourcesHealth.ts [hunk 1]
```diff
@@ -9,13 +9,19 @@ export interface ResourcesHealth {
 
 /**
  * "Failed to deploy" is mapped to resourceSummary.lastHandlerRun.failed — a handler execution
- * failure is the most direct read of that phrase.
+ * failure is the most direct read of that phrase. Severity stays keyed off that count alone;
+ * the total resource count and non-compliant count are shown as extra context, not additional
+ * severity triggers.
```
Comment (LukasStordeur) [inline src/Slices/DashboardV2/UI/resourcesHealth.ts:14]: Need to confirm this, if non-compliant is considered as an Attention warning, or can remain as Healthy.

File: src/Slices/DashboardV2/UI/Components/Resources/ResourcesCard.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+import { DependencyContext } from "@/UI/Dependency";
+import { SearchHelper } from "@/UI/Routing/SearchHelper";
+import { words } from "@/UI/words";
+import { MINIMAL_PAGE } from "../../EnvironmentHealthRow";
+import { IconBadge } from "../IconBadge";
+import { ResourceStatusBar } from "./ResourceStatusBar";
+
+const searchHelper = new SearchHelper();
+
+const EMPTY_COMPLIANCE: Record<Resource.ComplianceKey, number> = {
```
Comment (AronH99) [inline src/Slices/DashboardV2/UI/Components/Resources/ResourcesCard.tsx]: Except for the ts issue which can probably be solved in another way i don't think we really need these default fallbacks, I tested it with empty environment and as i though the bars will still show a grayed out bar with 0 in it..
So I think I would delete these?

File: src/Slices/DashboardV2/UI/Components/Resources/ResourceStatusBar.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+              </FlexItem>
+            </Flex>
+          ))}
+        </Flex>
+      )}
+    </Flex>
+  );
+};
+
+const Dot = styled.span`
```
Comment (AronH99) [inline src/Slices/DashboardV2/UI/Components/Resources/ResourceStatusBar.tsx]: think this is a duplicate of an existing Dot component you already made for the other cards

File: src/Slices/DashboardV2/UI/resourcesHealth.ts (hunk 1 above)
Comment (AronH99) [reply src/Slices/DashboardV2/UI/resourcesHealth.ts:14]: Already confirmed or no?

Comment (LukasStordeur) [reply src/Slices/DashboardV2/UI/resourcesHealth.ts:14]: Will need Bart for that, so next week. I think we can leave it as is for now, and tweak it later

**Review by AronH99 — APPROVED** [review]
